content/utmt/rooms/creating new rooms/design/_index.md

```diff
@@ -0,0 +1,76 @@
+Take note of the depth variable when adding tiles. Since UTMT does not have a layer system, this value is what decides which tiles are in the foreground and which are in the background, with lower values being shown in front of higher values.
+
+{{<hint type="tip">}}
+Here are some depth values to take note of:
+- Backgrounds: 100
+- Samus: -1
+- Foreground: -100
+- Breakable Blocks: -111/-112
+- Fading Tiles: -120 (Do not work without an `oFadeTiles` object)
+{{</hint>}}
+
+Something you can use to make the tiling process a lot faster is to copy existing tiles by holding alt and dragging them around the room. This will copy everything from that tile, including depth and tileset. This drastically speeds up the tiling process.  
+It is recommended to start with the tiles that the player will actually interact with, then adding stuff outside the playable area and background decorations.
+
+{{<columns>}}
+{{< img name="initialTiling" size=origin >}}
+<--->
+{{< img name="oobTiling" size=origin >}}
+<--->
+{{< img name="decorationTiling" size=origin >}}
+{{</columns>}}
+
+Now we can begin adding collision, for this we will use objects, notably `oSolid1` and `oSlope(1-4)`. On the left `Filter by name...` box, search for `Solid`. Here you will find all types of Solids, though for most purposes, `oSolid1` will be all you need. You can add it just by dragging the object into the room and placing it where you want it, then you can scale it to fit your needs. Then, for sloped surfaces you can use the `oSlope(1-4)` and `oSlope(1-4)B`. Don't try to rotate these objects, instead, if the slope doesn't have the correct orientation, take a slope with a different number and drag it into the definition box of your selected slop until you find the one with the right orientation.
+{{<hint type="warning">}}
+Never Delete objects, or basically anything in UTMT. Everything in UTMT is organized in a large list, with the newest addition always located in the bottom. Deleting something that is not literally the last thing in this list can have unforseen consequences and will often break things. If you mistakenly put the wrong object in your room, replace it with another object that you actually need (Just drag an object from into the definition box of your misplaced object)
+{{</hint>}}
+
+For puzzles and other uses, you can also search for `missile`, `bomb`, `super missile` and other destroyable blocks. Keep in mind that these blocks, as well as solid blocks, are invisible and do not appear without putting tiles over them or giving them creation code.  
+For this, there are two solutions:
+1. Add creation code for each of them and use `tile_link(tileset,x,y)` as its content. It will link the block to a tileset and will automatically use a 16 x 16 size. The x and y coordinates correspond with the top-left corner of the breakable block. 
```

Comment:
```suggestion
1. Add creation code for each of them and use `tile_link(tileset,x,y)` as its content. It will link the block to a tileset and automatically use a 16 x 16 size. The x and y coordinates correspond with the top-left corner of the breakable block. 
```